src/types.rs

```diff
@@ -335,6 +340,8 @@ pub struct OrderState {
     pub finalize: String,
     /// The certificate URL, which becomes available after finalization
     pub certificate: Option<String>,
+    /// The certificate that this order is replacing
+    pub replaces: Option<EncodedCertificateIdentifier>,
```

Comment:
FWIW: calling out that this will be a breaking change but I haven't bumped the crate version yet in case you want to approach it a different way.

Reply:
Should we make this `#[non_exhaustive]` while we're at it? I think this is pretty much an output-only type, so I think that could make sense?